spec.md

```diff
@@ -2649,7 +2747,8 @@ type_ground = "Clock" | "Reset" | "AsyncReset"
 type_aggregate = "{" , field , { field } , "}"
                | type , "[" , int , "]" ;
 field = [ "flip" ] , id , ":" , type ;
-type = type_ground | type_aggregate ;
+consttype = "const", type_ground | type_aggregate
+type = type_ground | type_aggregate | consttype ;
```

Comment:
Nit: This can be simplified to the following if `consttype` is not intended to be used elsewhere (and I don't think it makes sense to try to put the `AsyncReset` reg _semantic_ into EBNF which is the only place I can think of where `consttype` could be used):

```suggestion
type = [ "const" ] , ( type_ground | type_aggregate );
```